### PR TITLE
About section

### PR DESCRIPTION
**Pull request by ameseee**
## What's this PR do?

This PR 
- improves the styling of the About section on my app. 
- Adds the counter
- Adds the call to action section at bottom 

## Pre-Merge TODOs
 - [x] Screenshots Included
    - ANY change to the UI of the app must have screenshots/gifs
    - Hint: recordit is really great for this
 - [x] 2 PR reviews completed

## Learning

I had to read up on CSS grid and line heights. I had to read about different ways to implement the counter. 

## Where should the reviewer start?

About.js
About.scss

## Screenshots (if appropriate)
image
image

image
image
image

### CODE REVIEW

File: src/components/about/AboutStyles.css [hunk 1]
```diff
@@ -133,9 +144,101 @@
   align-items: center;
   justify-content: center; }
 
+.report-person-icon {
```
Comment (francylang) [inline src/components/about/AboutStyles.css:147]: CSS game is strong.

**Review by francylang — APPROVED** [review]

File: src/components/about/About.js [hunk 2]
```diff
@@ -1,111 +1,153 @@
 import React, { Component } from 'react';
-import { VictoryPie, VictoryChart } from 'victory';
```
Comment (jenPlusPlus) [inline src/components/about/About.js:2]: really nice use of charts!

**Review by jenPlusPlus — APPROVED** [review]
This looks SO clean and professional! great job!

**Pull request merged**